src/bin/order_and_cancel.rs

```diff
@@ -55,4 +57,33 @@ async fn main() {
     // This response will return an error if order was filled (since you can't cancel a filled order), otherwise it will cancel the order
     let response = exchange_client.cancel(cancel, None).await.unwrap();
     info!("Order potentially cancelled: {response:?}");
+
+    // Order and Cancel with cloid
+    let cloid = Uuid::new_v4();
```

Comment:
nit: could you separate the cloid example into a separate file, e.g. `order_and_cancel_with_cloid.rs`. This follows the pattern we have for the python-sdk, i.e. `basic_order_with_cloid.py`, in https://github.com/hyperliquid-dex/hyperliquid-python-sdk/tree/master/examples

Reply:
can do